hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/DefaultHiveRecordMerger.java

```diff
@@ -39,33 +41,33 @@ public class DefaultHiveRecordMerger extends HoodieHiveRecordMerger {
   private String[] orderingFields;
 
   @Override
-  public Option<Pair<HoodieRecord, Schema>> merge(HoodieRecord older, Schema oldSchema, HoodieRecord newer, Schema newSchema, TypedProperties props) throws IOException {
+  public Pair<HoodieRecord, Schema> merge(HoodieRecord older, Schema oldSchema, HoodieRecord newer, Schema newSchema, TypedProperties props) throws IOException {
     ValidationUtils.checkArgument(older.getRecordType() == HoodieRecord.HoodieRecordType.HIVE);
     ValidationUtils.checkArgument(newer.getRecordType() == HoodieRecord.HoodieRecordType.HIVE);
     if (newer instanceof HoodieHiveRecord) {
       HoodieHiveRecord newHiveRecord = (HoodieHiveRecord) newer;
       if (newHiveRecord.isDelete(newSchema, props)) {
-        return Option.empty();
+        return Pair.of(new HoodieEmptyRecord<>(newer.getKey(), HoodieOperation.DELETE, OrderingValues.getDefault(), HoodieRecord.HoodieRecordType.HIVE), newSchema);
```

Comment:
why not takes the ordering value from the original record instead of `OrderingValues.getDefault()`?

Reply:
I also noticed this gap some where else. 
if this is not breaking existing behavior, can we file a follow up jira and revisit entire deletes + ordering value gaps separately rather than trying to fix it in the same patch.

Reply:
That is because the deletes are pre handled by the buffered record merger itself. But in this patch, the pre-handling is removed.